modules/ebi-metagenomics/dbcan/easysubstrate/meta.yml

```
@@ -51,22 +51,22 @@ input:
           - meta
         description: Type of sequence input. protein=proteome; prok=prokaryote; meta=metagenome
 output:
-  - overview_output:
+  - overview_txt:
       - meta:
           type: map
           description: "Groovy Map containing sample information"
-      - results/${prefix}_overview.txt.gz:
+      - results/${prefix}_overview.tsv.gz:
           type: file
           description: "Summarizes CAZyme predictions across tools, including SignalP results."
-          pattern: "results/*_overview.txt.gz"
-  - dbsub_output:
+          pattern: "results/*_overview.tsv.gz"
+  - dbsub_output_tsv:
       - meta:
           type: map
           description: "Groovy Map containing sample information"
-      - results/${prefix}_*.hmm.out.gz:
+      - results/${prefix}_dbcan_hmm_results.tsv.gz:
           type: file
           description: "Output from the dbCAN_sub run."
-          pattern: "results/*.hmm.out.gz"
+          pattern: "results/*_dbcan_hmm_results.tsv.gz"
   - diamond_output:
```

```suggestion
  - dbhmm_output_tsv:
      - meta:
          type: map
          description: "Groovy Map containing sample information"
      - results/${prefix}_*.hmm.out.gz:
      - results/${prefix}_dbcan_hmm_results.tsv.gz:
          type: file
          description: "Output from the dbCAN_hmm run."
          pattern: "results/*.hmm.out.gz"
          pattern: "results/*_dbcan_hmm_results.tsv.gz" 
  - diamond_output:
```
Changed the original hmm output to be for the sub hmm output and added this for the hmm one